src/artm/core/dictionary.h

```diff
@@ -46,7 +46,7 @@ class Dictionary {
   std::vector<DictionaryEntry> entries_;
   std::unordered_map<Token, int, TokenHasher> token_index_;
   std::unordered_map<int, Token> index_token_;
-  std::unordered_map<int, std::unordered_map<int, int> > cooc_values_;
+  std::unordered_map<int, std::unordered_map<int, std::vector<float> > > cooc_values_;
```

Comment:
Ok, I thought that
- all types of coherence fit simple formula (1), e.g. calculate an average across of VALUE(w_i, w_j) for all pairs in a specific list (like top tokens, or topic kernels). 
- and all complaxity is hidden in a way VALUE(w_i, w_j) is calculated. So the plan was to push this complexity to the user, and keep BigARTM clean :)